### ISSUE TITLE
error on building 

### ISSUE THREAD

**Issue by whatSocks** [RadicalxChange/rxc-voice#33]
I ran `docker-compose -f docker-compose-polis.yml build` (the command suggested in readme didn't work for me, perhaps a separate issue, what I did is documented [here](https://github.com/RadicalxChange/rxc-voice/pull/34)) and I ran into the following:

```
[+] Building 128.8s (10/10) FINISHED
 => [internal] load build definition from Dockerfile                                                                                                                  0.0s
 => => transferring dockerfile: 176B                                                                                                                                  0.0s
 => [internal] load .dockerignore                                                                                                                                     0.0s
 => => transferring context: 34B                                                                                                                                      0.0s
 => [internal] load metadata for docker.io/library/node:latest                                                                                                        1.4s
 => [internal] load build context                                                                                                                                     0.0s
 => => transferring context: 4.56kB                                                                                                                                   0.0s
 => [1/6] FROM docker.io/library/node@sha256:25516f3de85ebf588e29d81052495d2e1177b55cddbd7ddab2f5ff2c4496dd5e                                                         3.9s
 => => resolve docker.io/library/node@sha256:25516f3de85ebf588e29d81052495d2e1177b55cddbd7ddab2f5ff2c4496dd5e                                                         0.0s
 => => sha256:aac9e540d3ed4f1037ad7ec712455a17d425b70ce5b49416c65f986bca6a7ba3 4.20kB / 4.20kB                                                                        0.1s
 => => sha256:25516f3de85ebf588e29d81052495d2e1177b55cddbd7ddab2f5ff2c4496dd5e 1.21kB / 1.21kB                                                                        0.0s
 => => sha256:2b67c5888e26a4427faec4741c9db1445df567c3bd76bc35c59a39f83ba75ba1 2.21kB / 2.21kB                                                                        0.0s
 => => sha256:4e45757feca1dbc5a0f9dbf42846c16c95d1d698968180baa70d4976aaf56043 34.34MB / 34.34MB                                                                      2.0s
 => => sha256:b670aa00baa822dad4be467368ab166a0db2322a8c99c163673253547562882f 2.30MB / 2.30MB                                                                        0.7s
 => => sha256:6817534de6bde9d1425d176fa9c71177eec84b5e4269c0b92a6f71a8edb82567 7.79kB / 7.79kB                                                                        0.0s
 => => extracting sha256:aac9e540d3ed4f1037ad7ec712455a17d425b70ce5b49416c65f986bca6a7ba3                                                                             0.1s
 => => sha256:227a1493d9c3838110553369e0621833fc0c9eea47ace5a5c0c627d545039654 284B / 284B                                                                            0.5s
 => => extracting sha256:4e45757feca1dbc5a0f9dbf42846c16c95d1d698968180baa70d4976aaf56043                                                                             1.5s
 => => extracting sha256:b670aa00baa822dad4be467368ab166a0db2322a8c99c163673253547562882f                                                                             0.1s
 => => extracting sha256:227a1493d9c3838110553369e0621833fc0c9eea47ace5a5c0c627d545039654                                                                             0.0s
 => [2/6] RUN mkdir /polis-app                                                                                                                                        0.3s
 => [3/6] COPY polis-app/ /polis-app                                                                                                                                  0.0s
 => [4/6] COPY scripts/ /scripts                                                                                                                                      0.0s
 => [5/6] WORKDIR /polis-app                                                                                                                                          0.0s
 => ERROR [6/6] RUN npm install                                                                                                                                     123.1s
------
```

**Comment by alexrandaccio** [comment]
Hey @whatSocks, thanks for your interest in the project! If I'm not mistaken, I believe this is an issue that I solved yesterday in PR #31. If you haven't pulled that commit to your local branch, I suggest you do so now.

The problem was caused by the [recent node release](https://nodejs.org/en/about/releases/) (16.0.0) that came out April 20th. This project requires node 15.14.0, so I tagged the docker containers with the correct version yesterday.
